cypress/integration/regression/decision-tree-regression-pt2.spec.ts

```diff
@@ -0,0 +1,255 @@
+  })
+
+  it('should open decision tree panel | step 24', () => {
+    datasetPage.visit()
+    datasetPage.leftPanel.leftPanelHeader.checkLabelText('Datasets')
+    datasetPage.leftPanel.datasetsListElem.getButtonByText(datasetName).click()
+    datasetPage.datasetInfo.datasetHeader.haveText(datasetName)
+    datasetPage.datasetInfo.openInViewer.click()
+    datasetPage.datasetInfo.viewerOption.contains('Decision Tree Panel').click()
+    cy.url().should('include', `/filter?ds=${datasetName}`)
+  })
+
+  it('should apply decision tree presert | step 25', () => {
+    datasetPage.visit(`/filter?ds=${datasetName}`)
+    decisionTreesPage.decisionTreeMenu.selectDecision.first().click()
+    cy.intercept('POST', '/app/statunits').as('decTreeUpload')
+    decisionTreesPage.decisionTreeMenu.selectDecision.getFilter(filterName)
+    decisionTreesPage.decisionTreeResults.stepCard.countElements(18)
+    cy.wait('@decTreeUpload')
+  })
+
+  it('should apply changes | step 26', () => {
+    datasetPage.visit(`/filter?ds=${datasetName}`)
+    decisionTreesPage.decisionTreeMenu.selectDecision.first().click()
+    cy.intercept('POST', '/app/statunits').as('decTreeUpload')
+    decisionTreesPage.decisionTreeMenu.selectDecision.getFilter(filterName)
+    decisionTreesPage.decisionTreeResults.stepCard.countElements(18)
+    cy.wait('@decTreeUpload')
+    decisionTreesPage.decisionTreeResults.addAttribute.first().click()
+    decisionTreesPage.attributesList.searchForAttr
+      .eq(0)
+      .type('Compound_Request')
+    decisionTreesPage.decisionTreeResults.graphHeaders.eq(0).click()
+    decisionTreesPage.decisionTreeResults.selectReset.select(
+      'Homozygous Recessive/X-linked',
+    )
+    decisionTreesPage.attributesList.replaceButton.click()
+    cy.wait('@decTreeUpload')
+    decisionTreesPage.decisionTreeResults.searchGraphResults
+      .eq(1)
+      .type('Compound_Request')
+    decisionTreesPage.decisionTreeResults.stepCard.countElements(1)
+  })
+
+  it('should create new decision tree | step 27', () => {
+    decisionTreesPage.visit(`/filter?ds=${datasetName}`)
+    decisionTreesPage.decisionTreeResults.addAttribute.click()
+    decisionTreesPage.attributesList.searchForAttr.eq(0).type('aller')
+    decisionTreesPage.decisionTreeResults.graphHeaders.eq(0).click()
+    decisionTreesPage.attributesList.selectAll.contains('Select All').click()
+    cy.intercept('POST', '/app/statunits').as('applyAttributes')
+    decisionTreesPage.attributesList.addSelectedAttributes.click()
+    cy.wait('@applyAttributes').its('response.statusCode').should('eq', 200)
+    decisionTreesPage.decisionTreeMenu.createNew.click()
+    decisionTreesPage.decisionTreeMenu.newDecisionTreeNameInput.type(
+      `${decisionTreeName}`,
+    )
+    cy.intercept('POST', '/app/dtree_stat').as('createNewTree')
+    decisionTreesPage.decisionTreeMenu.applyNewTree.click()
+    cy.wait('@createNewTree')
+    decisionTreesPage.decisionTreeResults.anyChangeAlert.checkExists()
```

Comment:
Let's check that item was actually created. We also could check alert's message.

Reply:
added check if the filter was created and check of the alert message